mutmut/__main__.py

```diff
@@ -587,20 +602,34 @@ def run_mutation_tests(config, mutations_by_file):
         run_mutation_tests_for_file(config, file_to_mutate, mutations)
 
 
-def read_coverage_data(use_coverage):
+def read_coverage_data():
     """
     :type use_coverage: bool
     :rtype: CoverageData or None
     """
-    if use_coverage:
-        print('Using coverage data from .coverage file')
-        # noinspection PyPackageRequirements,PyUnresolvedReferences
-        from coverage import Coverage
-        cov = Coverage('.coverage')
-        cov.load()
-        return cov.get_data()
-    else:
-        return None
+    print('Using coverage data from .coverage file')
+    # noinspection PyPackageRequirements,PyUnresolvedReferences
+    from coverage import Coverage
+    cov = Coverage('.coverage')
+    cov.load()
+    return cov.get_data()
+
+
+def read_patch_data(use_patch_file):
+    print('Using patch data from ' + use_patch_file)
+    try:
+        # noinspection PyPackageRequirements
+        import whatthepatch
+    except ImportError:
+        print('The --use-patch feature requires the whatthepatch library. Run "pip install whatthepatch"')
+        return -1
```

Comment:
We should just re-raise the exception. 
```python

 try:
    # noinspection PyPackageRequirements
    import whatthepatch
except ImportError:
    print('The --use-patch feature requires the whatthepatch library. Run "pip install whatthepatch"')
    raise
```


On an python exception fully raising and terminating execution a return code of `1` should be set. This conforms to our exit code standard of 1 being some error within the execution of mutmut. Also the user then gets the full traceback on the import error, this could be useful for debugging if the import error is a fault on the users end (e.g. corrupt python environment).